Dockerfile

```diff
@@ -44,6 +43,8 @@ COPY --from=0 /go/bin/dex /usr/local/bin/dex
 
 USER 1001:1001
 
+WORKDIR /
```

Comment:
`WORKDIR /` is completely no-op, because you already are in the `/` directory

Reply:
You also have a typo in commit message:
```patch
-chore(docke)r: move workdir down in the final image
+chore(docker): move workdir down in the final image
```

Reply:
Thanks!